config/redirects.yml

```diff
@@ -337,7 +337,7 @@ redirects:
   "/urgent-call-for-qualified-teachers": "/returning-to-teaching"
 
   # Blog pages
-  "/blog/5-reasons-to-attend-a-train-to-teach-event": "/blog/5-reasons-to-attend-a-get-into-teaching-event"
+  "/blog/5-reasons-to-attend-a-train-to-teach-event": "/events/about-get-into-teaching-events"
```

Comment:
sorry - I think my previous comment was confusing - we need **both** redirects

"/blog/5-reasons-to-attend-a-train-to-teach-event": "/events/about-get-into-teaching-events"
"/blog/5-reasons-to-attend-a-get-into-teaching-event": "/events/about-get-into-teaching-events"

Reply:
Ah sorry. No problem. I'll update that asap.